ansible-role-requirements.yml

```
@@ -155,9 +155,9 @@
   trackbranch: stable/ussuri
 - name: os_neutron
   scm: git
-  src: https://opendev.org/openstack/openstack-ansible-os_neutron
-  version: bd74c76dfb5ed5e0d1a01ee6e1efd1bdc7871371
-  trackbranch: stable/ussuri
+  src: https://github.com/elastx/openstack-ansible-os_neutron
+  version: cc49ed000f000b39895c9d0e0c84e87b5f9f6ce2
```

Updated branch to fix accidentally editing previous commits, new commit ID:
```suggestion
  version: 270e6c3a1c85df3a0bf73a1e99553ca891300792
```